evaluation/human_evaluation/utils/utils.py

```diff
@@ -0,0 +1,181 @@
+import streamlit as st
+from google.oauth2.service_account import Credentials
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from dotenv import load_dotenv
+from typing import List, Dict, Any
+
+import os
+
+load_dotenv()
+
+SERVICE_ACCOUNT_FILE = os.getenv("GOOGLE_CREDENTIALS_JSON")
+GOOGLE_FORM_ID = os.getenv("GOOGLE_FORM_ID")
+GOOGLE_SHEET_ID = os.getenv("GOOGLE_SHEET_ID")
+
+SCOPES = [
+    "https://www.googleapis.com/auth/forms.body",
+    "https://www.googleapis.com/auth/forms.responses.readonly",
+    "https://www.googleapis.com/auth/spreadsheets",
+]
+
+creds = Credentials.from_service_account_file(SERVICE_ACCOUNT_FILE, scopes=SCOPES)
+sheets_service = build("sheets", "v4", credentials=creds)
+forms_service = build("forms", "v1", credentials=creds)
+
+
+def parse_custom_input(custom_input: str, max_value: int) -> List[int]:
+    """
+    Parse custom input for row ranges and return a sorted list of row numbers.
+
+    Args:
+    - custom_input (str): A string representing the custom input for row ranges.
+    - max_value (int): The maximum row number available.
+
+    Returns:
+    - List[int]: A sorted list of row numbers.
+    """
+    result: List[int] = []
+    ranges = custom_input.split(",")
+    for r in ranges:
+        if "-" in r:
+            start, end = r.split("-")
+            start = int(start) if start else 2
+            end = int(end) if end else max_value + 1
+            result.extend(range(start, end + 1))
+        else:
+            result.append(int(r))
+    return sorted(set(result))
+
+
+def selected_questions(
+    questions: List[Dict[str, Any]], parsed_values: List[int]
+) -> List[Dict[str, Any]]:
```

Comment:
You can use lowercase `dict` as type too.

Reply:
Yeah, it was dict before, I just changed it to Dict for backward compatibility.